sccm/core/clients/manage/manage-clients.md

```diff
@@ -298,10 +298,13 @@ Adjust the size of the client cache without having to reinstall the client by co
 
 1.  Open a Windows command prompt and change the folder to the location in which CCMSetup.exe is located.  
 
-2.  Type **Ccmsetup.exe /uninstall**, and then press **Enter.**  
+2.  Type **CCMSetup.exe /uninstall**, and then press **Enter.**  
 
 > [!NOTE]  
->  The uninstall process displays no results on the screen. To verify that client uninstallation has succeeded, examine the log file **CCMSetup.log** in the folder *%windir%\ ccmsetup* on the client computer.  
+>  The uninstall process displays no results on the screen. To verify that client uninstallation has succeeded, examine the log file **CCMSetup.log** in the folder *%windir%\ccmsetup\logs* on the client computer.  
+
+> [!TIP]
+> If you need to wait for the uninstall to complete before doing something else, you can execute `Wait-Process CCMSetup` in PowerShell to halt until the process completes.

```

Comment:
I tried to reword this tip a bit. Also, we don't use the word [execute](https://docs.microsoft.com/style-guide/a-z-word-list-term-collections/e/execute) 🙉 

```suggestion
> If you need to wait for the uninstall process to complete before doing something else, run `Wait-Process CCMSetup` in PowerShell. This command can pause a script until the CCMSetup process completes.
```